snorkel/utils.py

```diff
@@ -55,6 +55,14 @@ def camel_to_under(name):
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', name)
     return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
 
+def sparse_nonzero(X):
+    """Sparse matrix with value 1 for i,jth entry !=0"""
+    X_nonzero = X.copy()
+    if sparse.isspmatrix_csr(X) or sparse.isspmatrix_csc(X):
+        X_nonzero.data[X_nonzero.data != 0] = 1
+    else:
+        raise ValueError("Only supports CSR/CSC matrices")
+    return X_nonzero
```

Comment:
Can you quickly add in the rest of the support just mirroring the below code (for dense and LIL matrices)?  This will be:
- Dense: `X_nonzero[X_nonzero != 0] = 1`
- LIL: `X_nonzero.data = [np.ones(len(L)) for L in X_nonzero.data]`